fv3core/stencils/fv_dynamics.py

```diff
@@ -43,15 +45,15 @@ def init_pfull(
 
 def compute_preamble(
     state,
-    grid,
-    namelist,
+    is_root_rank: bool,
+    config: DynamicalCoreConfig,
     fv_setup_stencil: FrozenStencil,
     pt_adjust_stencil: FrozenStencil,
 ):
-    if namelist.hydrostatic:
+    if config.hydrostatic:
```

Comment:
similar comment. when it's called namelist, you immediately know where it originated from. when it's config you have some digging to do. I know the intention is to support other modes of running the model that don't necessarily have a file on disk, but still seems more user-friendly to adopt some naming conventions to orient the people transitioning from Fortran.

Reply:
Thanks. If it helps, Lucas did say it makes sense to rename it away from "namelist", and that it is a Fortran-specific term which doesn't apply in other contexts.

Reply:
I also imagined that, but what Lucas told me is that calling a configuration yaml file and configuration classes a "namelist" could be confusing to a Fortran programmer. He said he supports naming it something else, and gave a thumbs up to "config".